PreFreeSurfer/PreFreeSurferPipeline.sh

```diff
@@ -748,7 +996,24 @@ ${RUN} ${HCPPIPEDIR_PreFS}/AtlasRegistrationToMNI152_FLIRTandFNIRT.sh \
   --ot2=${AtlasSpaceFolder}/${T2wImage} \
   --ot2rest=${AtlasSpaceFolder}/${T2wImage}_restore \
   --ot2restbrain=${AtlasSpaceFolder}/${T2wImage}_restore_brain \
-  --fnirtconfig=${FNIRTConfig} 
-
+  --fnirtconfig=${FNIRTConfig} \
+  --brainextract=${BrainExtract}
 log_Msg "Completed!"
+}
+
+main () {
```

Comment:
I know Matt doesn't like lots of functions, and having `main()` be a function doesn't really serve a purpose (in the previous `opts.shlib`, it was used only to support both named options and bare arguments).  I would consider putting conditionals testing RunMode around each section (you can do `if ((RunMode < 3))` and it will do what you think it should) rather than making each section into a function (making it obvious that the order that things get run is in fact what it looks like, and making copy-paste debugging easier).